index.rst

```diff
@@ -149,6 +150,22 @@ To ensure that other developers are aware of the pending change, post to the app
 
 .. _Community: https://community.lsst.org
 
+Merge to master and move packages on github
+-------------------------------------------
+
+Once your review is complete and you've announced the move as described above, you can merge your branches to master. But merging to master isn't the final step.
+
+All LSST dependencies need to live in the `lsst` github organization. In order to keep our build system happy, you will also need to move ``meas_worst`` and ``pipe_best`` between organizations via the `github transfer repository`_ instructions. Because github puts in an automatic redirect as part of those instructions, people with clones pointing to the pre-move repositories will not have to change their clone `remote` links. You may need the help of a senior developer/manager in order to complete this move step, as the person doing each move needs to be an administrator of both the source and destination organziation.
```

Comment:
It's not the build system that cares, it's the system that does releases that cares.